package/samplers/fcmaes/README.md

```diff
@@ -0,0 +1,56 @@
+---
+author: Jacob Pfeil
+title: Optuna Wrapper for Fast CMA-ES
+description:
+tags: [sampler, cmaes, fcmaes]
+optuna_versions: [4.5.0]
+license: MIT License
+---
+
+## Abstract
+
+C++ optimized implementation of CMA-ES using the fcmaes library (https://github.com/dietmarwo/fast-cma-es)
+
```

Comment:
Could you add a section about installation/dependencies like https://hub.optuna.org/samplers/catcma/?
This package requires `fcmaes`.

```
pip install fcmaes
```